gcloud/datastore/connection.py

```diff
@@ -317,9 +304,9 @@ def commit(self, project, request_pb):
         request_pb.project_id = project
         try:
             return self._stub.Commit(request_pb, GRPC_TIMEOUT_SECONDS)
```

Comment:
> What's attracting you to passing your `grpc.CallCredentials` value at each RPC invocation?

Nothing. I was just confusing that with the credentials classes with `oauth2client`